src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let finalResult = 0;
+
+  const adder = (number) => {
+    if (!number && number !== 0) {
```

Comment:
There is more simple option for this check.
```suggestion
    if (number === undefined) {
```